cli/cli.go

```diff
@@ -32,6 +33,11 @@ func Run() {
 			Value: "info",
 			Usage: fmt.Sprintf("Log level (options: debug, info, warn, error, fatal, panic)"),
 		},
+
+		cli.BoolFlag{
+			Name:  "experimental",
```

Comment:
Why not use a flag like `autoreschdule`? You can state experimental in flag description. `experimental` is not one feature. Over time this flag would be used for other features. It's not backward compatible.